app/controllers/api/scores_controller.rb

```diff
@@ -5,7 +5,7 @@ class ScoresController < ApplicationController
     before_action :validate_score_user_id, only: :destroy
 
     def user_feed
-      scores = Score.all.order(played_at: :desc, id: :desc)
+      scores = Score.all.includes(:user).order(played_at: :desc, id: :desc)
```

Comment:
You don't need to fix that here. https://github.com/gabrielsorin88/golfr_backend/pull/1 will fix the n+1 query problem as soon as it is merged.